src/js/dialogs/sandbox.js

```diff
@@ -727,56 +727,21 @@ exports.dialog = {
       markdowns: [
         '## Witaj w Learn Git Branching!',
         '',
-        'Celem tej aplikacji jest zilustrowanie rozległych i złożonych relacji pomiędzy procesami związanymi z pracą z GIT-em.',
-        'Mam nadzieję, że ci się to spodoba i może nawet się czegoś nauczysz!',
-        '"Learn Git Branching" to najbardziej wizualny i interaktywny sposób na naukę GIT-a w sieci.',
-        'Będziesz miał do czynienia z ekscytującymi poziomami wraz z pokazaniem krok po kroku potężnych funkcji, a może nawet z odrobiną zabawy po drodze.',
+        'Jesteś zainteresowany nauką Gita? Cóż, trafiłeś we właściwe miejsce!',
+        '"Learn Git Branching" jest najbardziej wizualnym i interaktywnym sposobem na naukę gita w sieci.',
+        'Czekają na Ciebie ekscytujące poziomy, demonstracje zaawansowanych funkcji krok po kroku. Może nawet będziesz się dobrze bawić.',
         '',
+        'Po tym oknie dialogowym zobaczysz różnorodność poziomów, które mamy do zaoferowania.',
         'Jeśli jesteś początkujący, po prostu zacznij od pierwszego poziomu.',
-        'Jeśli znasz już podstawy GIT-a, wypróbuj niektóre z naszych późniejszych, bardziej wymagających poziomów.',
+        'Jeśli znasz już podstawy gita, wypróbuj niektóre z naszych późniejszych, bardziej wymagających poziomów.',
         '',
-        '# Demo!',
-        '',
-        'Jeśli nie widziałeś tej demonstracji, możesz ją obejrzeć tutaj:',
-        '',
-        '[https://pcottle.github.io/learnGitBranching/?demo](https://pcottle.github.io/learnGitBranching/?demo)',
-        '',
-        'Masz dość tego okna? Dodaj `?NODEMO` do adresu URL, aby się go pozbyć, w ten sposób:',
-        '',
-        '[https://pcottle.github.io/learnGitBranching/?NODEMO](?NODEMO)'
-      ]
-    }
-  }, {
-    type: 'ModalAlert',
-    options: {
```

Comment:
so this modal alert left?

Reply:
I saw en_US translation without that part, so I removed it thinking it's not actual anymore.
Would you like me to bring it back?

Reply:
Oh then that's totally fine -- sorry I didn't check if it was more consistent or not.

Thanks for all your contributions on the polish translation!